jina/peapods/runtimes/zmq/zed.py

```diff
@@ -171,6 +171,15 @@ def _post_hook(self, msg: 'Message') -> 'ZEDRuntime':
         msg.update_timestamp()
         return self
 
+    @staticmethod
+    def _parse_params(parameters: Dict, executor_name: str):
+        parsed_params = parameters
+        specific_parameters = parameters.get(executor_name, None)
+        if specific_parameters:
+            parsed_params.update(**specific_parameters)
+            del parsed_params[executor_name]
```

Comment:
Why deleting?

Reply:
I did not have a clear view. I found it more clear to have a single source of truth, but can remove

Reply:
Should I change this @maximilianwerk , it may be more conservative yes

Reply:
Yes, please don't delete it.

Reply:
Done!